web/packages/teleterm/src/ui/Documents/KeyboardShortcutsPanel.tsx

```diff
@@ -67,20 +67,26 @@ export function KeyboardShortcutsPanel() {
 }
 
 function Entry(props: { title: string; shortcut: string }) {
+  const theme = useTheme();
   return (
     <>
       <Text textAlign="right" color="light" typography="subtitle1" py="4px">
         {props.title}
       </Text>
-      <MonoText bg="primary.main" textAlign="left" px="12px" py="4px">
+      <MonoText
+        style={{ fontFamily: theme.fonts.unsanitizedMono }}
```

Comment:
This can be reverted back to its original form as we want to use the configurable font only in the terminal, right?

Reply:
Ah yes, sorry.